templates/server-acl-init-job.yaml

```diff
@@ -101,6 +101,10 @@ spec:
                 -resource-prefix=${CONSUL_FULLNAME} \
                 -k8s-namespace={{ .Release.Namespace }} \
 
+                {{- if (and .Values.connectInject.enabled .Values.connectInject.healthChecks.enabled) }}
+                -enable-health-checks-controller=true \
```

Comment:
Out of curiosity, do we have to set this value to be `true`? Can we just pass `-enable-health-checks-controller` and the presence of the flag imply truthiness?

Reply:
pretty sure this is doable! when I refactor with the new naming I'll make this the case :)